src/computational_graph/graph.jl

```diff
@@ -386,22 +389,33 @@ end
 # Example: 
 ```julia-repl
 julia> g = propagator(𝑓⁺(1)𝑏⁺(2)𝜙(3)𝑓⁻(1)𝑏⁻(2))
-1: propagtor graph from f⁺(1)b⁺(2)ϕ(3)f⁻(1)b⁻(2)
+1: propagator graph from f⁺(1)b⁺(2)ϕ(3)f⁻(1)b⁻(2)
 
 julia> standardize_order!(g)
 
 julia> g, g.factor
-(1: propagtor graph from f⁻(1)b⁻(2)ϕ(3)b⁺(2)f⁺(1), -1.0)
+(1: propagator graph from f⁻(1)b⁻(2)ϕ(3)b⁺(2)f⁺(1), -1.0)
 ```
 """
 function standardize_order!(g::Graph)
```

Comment:
I agree with Kun's point that we should implement standardize_order! without introspecting on `node.type`.
Because all leafs are propagators while all other nodes involve larger CompositeOperators, we can just call correlator_order `if node in Leaves(g)`, and `normal_order` otherwise. Or, maybe normal order everything on first pass, then call `correlator_order` on the leaves after. Thoughts?

Reply:
Agreed. I have updated call correlator_order if node in Leaves(g), and normal_order otherwise.